e2etests/web/regular_integration_tests/test_driver/platform_messages_integration.dart

```diff
@@ -28,15 +28,15 @@ void main() async {
     await tester.tap(find.byKey(const Key('input')));
     // Focus in input, otherwise clipboard will fail with
     // 'document is not focused' platform exception.
-    html.document.querySelector('input').focus();
+    html.document.querySelector('input')?.focus();
```

Comment:
Makes sense, thanks!

Reply:
Unfortunatelly I'm getting the following error:
```
Target dart2js failed: Exception: test_driver/text_editing_integration.dart:43:29:
Error: Null safety features are disabled for this library.
    textFormField.controller!.text = 'New Value';
```
Looks like I will wait for these changes until the packet is moved to nnbd.

Reply:
In that case you don't need to change this line at all, because `querySelector('input').focus()` in old Dart is the same as `querySelector('input')!.focus()` in new Dart.

Reply:
Then I'm getting this exception in debug mode:
```
org-dartlang-app:/platform_messages_integration.dart:31:42: Error: Method 'focus' cannot be called on 'Element?' because it is potentially null.
 - 'Element' is from 'dart:html'.                                       
Try calling using ?. instead.                                           
    html.document.querySelector('input').focus();   
```

The same tests passes in profile mode, so we didn't need this change before.

Let me know if you have any suggestions.

Reply:
@jonahwilliams For these tests we are seeing conflicting results for different modes:

- debug mode has a nnbd error:
```
org-dartlang-app:/platform_messages_integration.dart:31:42: Error: Method 'focus' cannot be called on 'Element?' because it is potentially null.
 - 'Element' is from 'dart:html'.                                       
Try calling using ?. instead.                                           
    html.document.querySelector('input').focus();  
```
- profile/release modes do not recognize nnbd:
```
Target dart2js failed: Exception: test_driver/text_editing_integration.dart:43:29:
Error: Null safety features are disabled for this library.
```

Any ideas?

@dnfield have you seen it this issue for other e2e tests?